.github/workflows/interop_regression.yml

```diff
@@ -0,0 +1,313 @@
+      - name: delete namespace
+        if: always()
+        continue-on-error: true
+        run: kubectl delete ns --ignore-not-found=true --wait interop-${{ github.run_id }}-pixel-4
+
+
+  #  test-iphone-11:
+#    runs-on: ubuntu-latest
+#    needs: [vars, test-pixel-4]
+#    if: "!cancelled() && contains(fromJSON(needs.vars.outputs.devices), 'iphone-11')"
+#    steps:
+#      - uses: actions/checkout@v2
+#
+#      - name: get EKS access credentials
+#        run: aws eks update-kubeconfig  --name ${{ env.AWS_EKS_NAME }}
+#
+#      - name: run tests
+#        uses: ./.github/actions/run-tests
+#        with:
+#          namespace: interop-${{ github.run_id }}-iphone-11
+#          testbed: interop-04
+#          marker_expression: "${{ needs.vars.outputs.marker_overwrite || 'interop_uc_sanity and client_connect and ios' }}"
+#          configuration: "${{ secrets.LAB_CONFIGURATION }}"
+#          testing_docker_image: tip-tip-wlan-cloud-docker-repo.jfrog.io/cloud-sdk-nightly:${{ github.run_id }}
+#          additional_args: "-o model-iOS='iPhone-11' -o 'jobName=Github-Interop-iphone-11' -o 'jobNumber=${{ github.run_number }}' --skip-lanforge"
+#          allure_results_artifact_name: allure-results-iphone-11
+
+  report:
+    needs: [vars, test-pixel-4, test-galaxy-s20, test-galaxy-s10, test-galaxy-s9]
+    if: "!cancelled()"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        device: ${{ fromJson( needs.vars.outputs.devices ) }}
+
+    steps:
+      - name: checkout testing repo
+        uses: actions/checkout@v2
+
+      - uses: actions/download-artifact@v2
+        with:
+          name: allure-results-${{ matrix.device }}
+          path: allure-results
+
+      - name: download history of previous run
+        continue-on-error: true
+        run: |
+          LAST_RUN_ID=$(aws s3api head-object --bucket openwifi-allure-reports --key interop_regression/${{ matrix.device }}/latest/index.html | jq -r .Metadata.latest)
+          aws s3 cp --recursive s3://openwifi-allure-reports/interop_regression/${{ matrix.device }}/$LAST_RUN_ID/history history
+
+      - name: generate Allure report
+        uses: ./.github/actions/generate-allure-report
+        with:
+          results_path: ./allure-results
+          history_path: ./history
+          additional_metadata: |
+            Device=${{ matrix.device }}
+
+      - name: upload Allure report as artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: allure-report-${{ matrix.device }}
+          path: allure-report
+
+      # doing this to be able to aggregate multiple reports together later on
+      - name: copy results into report
+        run: cp -r allure-results allure-report/results
+
+      - name: upload to S3
+        if: github.ref == 'refs/heads/master' && needs.vars.outputs.marker_overwrite == ''
+        uses: ./.github/actions/allure-report-to-s3
+        with:
+          test_type: interop
```

Comment:
please change this to interop-regression

Reply:
Changed it